client/blocks/post-share/connection.jsx

```diff
@@ -50,7 +49,7 @@ const PostShareConnection = ( { connection, isActive, onToggle } ) => {
 			<div className="post-share__service-account-name">
 				<span>{ external_display }</span>
 			</div>
-			<FormToggle checked={ isActive } />
+			<ToggleControl checked={ isActive } />
```

Comment:
Should we open an a11y issue for this toggle not having a label?

Reply:
Good point. Opened #53002 for that.